crates/sozo/ops/src/migrate/mod.rs

```diff
@@ -61,12 +65,15 @@ where
     // Ideally, we want this rpc url to be exposed from the world.account.provider().
     rpc_url: String,
     guest: bool,
+    // Optional verification configuration
+    verification_config: Option<VerificationConfig>,
```

Comment:
See comments above to use an enum here, which has a variant where the verification is "disabled" to avoid the option and easily extend to other services.